templates/core/terraform/main.tf

```diff
@@ -80,21 +80,25 @@ module "appgateway" {
 }
 
 module "airlock_resources" {
-  source                            = "./airlock"
-  tre_id                            = var.tre_id
-  location                          = var.location
-  resource_group_name               = azurerm_resource_group.core.name
-  airlock_storage_subnet_id         = module.network.airlock_storage_subnet_id
-  airlock_events_subnet_id          = module.network.airlock_events_subnet_id
-  virtual_network_id                = module.network.core_vnet_id
-  enable_local_debugging            = var.enable_local_debugging
-  docker_registry_server            = var.docker_registry_server
-  mgmt_resource_group_name          = var.mgmt_resource_group_name
-  mgmt_acr_name                     = var.acr_name
-  api_principal_id                  = azurerm_user_assigned_identity.id.principal_id
-  arm_subscription_id               = var.arm_subscription_id
-  airlock_app_service_plan_sku_size = var.api_app_service_plan_sku_size
-  airlock_processor_subnet_id       = module.network.airlock_processor_subnet_id
+  source                                = "./airlock"
+  tre_id                                = var.tre_id
+  location                              = var.location
+  resource_group_name                   = azurerm_resource_group.core.name
+  airlock_storage_subnet_id             = module.network.airlock_storage_subnet_id
+  airlock_events_subnet_id              = module.network.airlock_events_subnet_id
+  virtual_network_id                    = module.network.core_vnet_id
+  enable_local_debugging                = var.enable_local_debugging
+  docker_registry_server                = var.docker_registry_server
+  mgmt_resource_group_name              = var.mgmt_resource_group_name
+  mgmt_acr_name                         = var.acr_name
+  api_principal_id                      = azurerm_user_assigned_identity.id.principal_id
+  arm_subscription_id                   = var.arm_subscription_id
+  airlock_app_service_plan_sku_size     = var.api_app_service_plan_sku_size
+  airlock_processor_subnet_id           = module.network.airlock_processor_subnet_id
+  airlock_servicebus                    = azurerm_servicebus_namespace.sb
+  applicationinsights_connection_string = module.azure_monitor.app_insights_connection_string
+  tre_core_tags                         = local.tre_core_tags
```

Comment:
shouldn't be var.tre_core_tags?

Reply:
no, this is the root module we we call the airlock module. tre_core_tags should be defined once as a local and then passed into child modules